Add Air Fresh VA4 description

McYars opened the pull request:
## Proposed change
Add Air Fresh VA4 description for pr76637



## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
<!--
    Details are important, and help maintainers processing your PR.
    Please be sure to fill out additional details, if applicable.
-->

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76637
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

dependabot[bot] committed:
Bump public_suffix from 4.0.7 to 5.0.0 (#23843)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

dependabot[bot] committed:
Bump public_suffix from 4.0.7 to 5.0.0 (#23843)

Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

jbouwh committed:
Improve MQTT template documentation link (#23848)

jbouwh committed:
Improve MQTT template documentation link (#23848)

frenck committed:
Add workflow to add new PRs to project (#23871)

Co-authored-by: Joakim Sørensen <joasoe@gmail.com>

frenck committed:
Add workflow to add new PRs to project (#23871)

Co-authored-by: Joakim Sørensen <joasoe@gmail.com>

McYars committed:
Add Air Fresh VA4 description

Add Air Fresh VA4 description for pr76637

McYars committed:
Add Air Fresh VA4 description

Add Air Fresh VA4 description for pr76637

MartinHjelmare commented:
Please rebase against `next` branch and target `next` branch with the PR instead.

frenck committed:
Revert "Add workflow to add new PRs to project" (#23873)

frenck committed:
Revert "Add workflow to add new PRs to project" (#23873)

McYars committed:
Merge branch 'home-assistant:current' into patch-1

McYars committed:
Merge branch 'next' into patch-1

McYars committed:
Merge branch 'patch-1' of https://github.com/McYars/home-assistant.io into pr/23872

McYars committed:
Revert "Bump public_suffix from 4.0.7 to 5.0.0 (#23843)"

This reverts commit eafc66f9476f15dffe346770c1b881422e2c2116.

McYars committed:
Revert "Improve MQTT template documentation link (#23848)"

This reverts commit a72761338e8c7b5679d3e98aeb4d4513bc4e62bf.

McYars committed:
Revert "Revert "Add workflow to add new PRs to project" (#23873)"

This reverts commit eecc53c44426ccf0120c8b4a5beed517537af27b.

McYars committed:
Revert "Revert "Revert "Add workflow to add new PRs to project" (#23873)""

This reverts commit f1dfc9d9dd8615e679e253fb237a442d6e1363af.

McYars commented:
> Please rebase against `next` branch and target `next` branch with the PR instead.

I changed branch and rebased to next, but how to initiate labels update?

MartinHjelmare commented:
There are unrelated commits in the branch now. It may be easier to start over in a fresh branch synced from `next` and apply your changes again. Then open a new PR.
